ALUM/ALUM/Components/FilledButton.swift

```diff
@@ -0,0 +1,57 @@
+//
+//  FilledButton.swift
+//  ALUM
+//
+//  Created by Yash Ravipati on 1/18/23.
+//
+
+import SwiftUI
+
+struct FilledInButton : ButtonStyle{
+    
+    @State var isLarge: Bool = false
+    @State var disabled: Bool = false
+    
+    var largeWidth = 358.0
+    var smallWidth = 104.0
+    
+    func makeBody(configuration: Configuration) -> some View {
+            if(!disabled){
+                configuration.label
+                    .frame(width: isLarge ? largeWidth : smallWidth)
```

Comment:
Giving this more thought, hardcoding widths for the buttons don't seem ideal. And given our use cases it will be best if we deal with their widths while using the component. 

Instead of hardcoding in the widths, let's use `.frame` to set the minWidth to 0 and maxWidth to .infinity. This will allow our button width to be resizable based on its parent.  If we do this, you can remove the width state of the button. 

For button heights, we don't have any other option but to hardcode it so feel free to do that. See if you can using minHeight and maxHeight can make our button more flexible but I'm not too worried about fixing the height.

Idea is that if you put two buttons with our styles inside an HStack, the buttons should resize well to co-exist in the same space. Lmk if you have any questions!